ethereum/sidecar/server.go

```diff
@@ -45,6 +46,19 @@ var (
 	// enough.
 	cachedEventsLimit = 500000
 
+	// bridgeOutLookBackPeriod is the look-back period used when fetching
+	// AssetsUnlocked events from the Mezo chain. It defines how far back we
+	// look when searching for unattested events.
+	bridgeOutLookBackPeriod = 30 * 24 * time.Hour // ~1 month
+
+	// assetsUnlockedBatchSize is the number of AssetsUnlocked events we can
+	// fetch from the Mezo blockchain in one gRPC call.
+	assetsUnlockedBatchSize = 100
+
+	// assetsUnlockedFetchingPeriod is the time period defining how often new
+	// AssetsUnlocked events should be fetched.
+	assetsUnlockedFetchingPeriod = 5 * time.Minute
```

Comment:
I would use 1 minute at most. This communication is cheap and Mezo has instant block finality. Why adding an artificial bottleneck in that case?